var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -347,3 +347,38 @@
 - command: systemctl disable google-accounts-daemon.service
   when:
     - platform == "gcp"
+
+#
+# We want the ssh service to start as early as possible during boot up,
+# so we explicitly remove all dependencies of the service here. We've
+# configured the service not to use the "default" dependencies, via the
+# "override.conf" file in the delphix-platform package. Unfortuantely
```

Comment:
Do you think it would be cleaner to pull all the changes in here instead of relying on both override.conf and an /etc/ file?

Reply:
I don’t. I considered that, but I prefer this two pronged approach so we can still use the override file for all things that can be solved with it. I think the override file is much easier to use/test/maintain, and more reliable, so I’d like to use it over ansible wherever possible.